modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -45,16 +45,13 @@ endif::[]
 |Cloned Disk
 |Provision source is a cloned disk.
 
-|Import
-|Import virtual machine from a supported provider.
-
 |Operating System
 |
-|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine.
```

Comment:
I'm not sure about this line about being available in the cluster. It's more the user selects the OS they want the vm to be optimized for because it's the one they are providing via the source.